x-pack/test/alerting_api_integration/packages/helpers/es_test_index_tool.ts

```diff
@@ -138,7 +144,16 @@ export class ESTestIndexTool {
       size: 1000,
       body,
     };
-    return await this.es.search(params, { meta: true });
+    const result = await this.es.search(params, { meta: true });
+    result.body.hits.hits = result.body.hits.hits.map((hit) => {
+      return {
+        ...hit,
+        // Easier to remove @timestamp than to have all the downstream code ignore it
+        // in their assertions
+        _source: omit(hit._source as Record<string, unknown>, '@timestamp'),
+      };
+    });
+    return result;
```

Comment:
In a rare case, we receive the document we expect 2nd in the list instead of first. Having a sort on timestamp will make sure it's consistent.